Sources/SwiftDocC/Model/Rendering/RenderContentCompiler.swift

```diff
@@ -163,7 +163,22 @@ struct RenderContentCompiler: MarkupVisitor {
             return [RenderInlineContent.text(link.plainText)]
         }
         
-        return [RenderInlineContent.reference(identifier: .init(resolved.absoluteString), isActive: true, overridingTitle: nil, overridingTitleInlineContent: nil)]
+        let linkTitleInlineContent = link.children.flatMap { visit($0) } as! [RenderInlineContent]
+        let plainTextLinkTitle = linkTitleInlineContent.plainText
```

Comment:
By calling `plainText`, I'm guessing we're losing formatting here? For example if you write `[_Title_](…)`, 'Title' won't be rendered in italics. That's a bit unfortunate but can be address as part of a future PR. It would be good to have an issue to track this.

Reply:
> I'm guessing we're losing formatting here?

Yes, I just conformed it.

> It would be good to have an issue to track this.

When this PR's fix plan is merged, I'll open a new issue to track the remaining issue left.